parameter/ParameterMgr.cpp

```diff
@@ -335,6 +147,7 @@ CParameterMgr::CParameterMgr(const string& strConfigurationFilePath, log::ILogge
     _pvLibRemoteProcessorHandle(NULL),
     _uiStructureChecksum(0),
     _pRemoteProcessorServer(NULL),
+    _commandParser(*this),
```

Comment:
further enhancement: only instantiate it if/when the remote interface is needed (e.g. don't create it if TuningAllowed=false).